t/test_repo_consistent.sh

```diff
@@ -26,10 +26,17 @@ for dir in "$@" ; do
 
 	while read -r line; do
 	    if [[ "$line" =~ $regex ]]; then
-	        echo "$known_destination_repos" | grep -q "${BASH_REMATCH[1]}\$" || { >&2 echo "Destination REPO file wasnt found in print_rsync_repo output {${BASH_REMATCH[1]}}   {$known_destination_repos}"; : $((++errs)); continue 2; }
+            # we excuse REPO_0 if print_rsync_iso has bsdtar (which is indication that REPO_0 comes from print_rsync_iso)
```

Comment:
s/excuse/exclude/?

Reply:
`excuse` was intentional, meaning that it is the only condition where we shouldn't show failure. Exclude may work here as well though, so you decide